samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -102,11 +106,15 @@ private TestRunner() {
     configs.put(JobCoordinatorConfig.JOB_COORDINATION_UTILS_FACTORY, PassthroughCoordinationUtilsFactory.class.getName());
     configs.put(JobCoordinatorConfig.JOB_COORDINATOR_FACTORY, PassthroughJobCoordinatorFactory.class.getName());
     configs.put(TaskConfig.GROUPER_FACTORY(), SingleContainerGrouperFactory.class.getName());
+    configs.put(JobConfig.JOB_DEFAULT_SYSTEM(), JOB_DEFAULT_SYSTEM);
```

Comment:
Does this need to be addConfig as well? Users might be setting a default system descriptor in their application themselves.

Reply:
if there is no usage of default system other than intermediate streams (when the user needs to set this), I can use addConfig